examples/README.md

```diff
@@ -57,7 +57,7 @@ This Scala example demonstrates how to use Riak Spark connector to do range quer
 Before running, you will need to create and activate a TS table called `ts_weather_demo` if it does not already exist. You can find more information on creating and activating TS tables [here](http://docs.basho.com/riak/ts/latest/using/creating-activating/), or you can run the following:
 
 ```
-riak-admin bucket-type create ts_weather_demo '{"props":{"n_val":3, "table_def":"CREATE TABLE ts_weather_demo (weather varchar not null, family varchar not null, time timestamp not null, temperature double, humidity double, pressure double, PRIMARY KEY ((weather, family, quantum(time, 1, 'h')), weather, family, time))"}}'
```

Comment:
@laurenrother from friday, when we were talking about formatting... when i changed the format to include capitalized keys, its worked, but it was becuase of ^^^^. I guess I deleted the non ascii key. So our discussion of whether the keys are case sensitive is inconclusive. But a quick check could figure this out.